src/main/java/com/milamber_brass/brass_armory/BrassArmory.java

```diff
@@ -25,27 +27,24 @@
 
     public BrassArmory() {
         // Register
-        IEventBus eventBus = FMLJavaModLoadingContext.get().getModEventBus();
-        BrassArmoryItems.register(eventBus);
-        BrassArmoryBlocks.register(eventBus);
+    	IEventBus eventBus = FMLJavaModLoadingContext.get().getModEventBus();
+    	//TODO: Move block renderers into the client package and their registration too!
+    	BrassArmoryBlocks.register(eventBus);
+    	BrassArmoryItems.register(eventBus);
         BrassArmoryEntityTypes.register(eventBus);
     }
 
     @SubscribeEvent
     public static void onCommonSetup(FMLCommonSetupEvent event) {
         LOGGER.debug("Running common setup.");
-
+        
         // Register custom dispenser behavior
-        CustomDispenserBehavior.init();
+        BrassArmoryDispenseBehaviors.init();
     }
 
     @SubscribeEvent
     public static void onClientSetup(FMLClientSetupEvent event) {
-        LOGGER.debug("Running client setup.");
-
-        // Register spear and arrow entity rendering handlers
-        RenderingRegistry.registerEntityRenderingHandler(BrassArmoryEntityTypes.SPEAR.get(), SpearEntityRenderer::new);
-        RenderingRegistry.registerEntityRenderingHandler(BrassArmoryEntityTypes.BA_ARROW.get(), BAArrowRenderer::new);
+        //Moved into ClientEventBusSubscriber
```

Comment:
Let's remove the entire onClientSetup method from the main file if we're not going to put anything here. Presuming the ClientEventBusSubscriber will handle all of this anyways, no point in keeping this one.

Reply:
Taking a closer look, let's just move the ClientEventBusSubscriber to the "client" package, then we can eliminate the "util" package entirely. Makes sense anyways, since it's client-handled stuff.

Reply:
Making changes and then removing this pull request until the weapons are actually ready to be merged.

Reply:
Done